task_8/task_8_10.py

```diff
@@ -0,0 +1,11 @@
+def chess():
+    coord = input('Input figure coordinates (example: d4): ')
+    if 104 < ord(coord[0]) or 57 > ord(coord[0])\
+            or 57 < ord(coord[1]) or 49 > ord(coord[1]):
+        return 'Out of chess field'
+    if ord(coord[0]) % 2 == 1 and ord(coord[1]) % 2 == 1:
+        return 'square is black'
+    return 'square is white'
```

Comment:
color = 'black' if ord(coord[0]) % 2 == 1 and ord(coord[1]) % 2 == 1 else 'white'
    return f'square is {color}'